### ISSUE TITLE
Colour contrast insufficient

### ISSUE THREAD

**Issue by rscano** [italia/forum.italia.it#7]
![Colour contrast 2,8:1... shall be 4,5:1 for number and 3:1 for text.](https://github.com/italia/forum.italia.it/assets/18532119/d3129ab4-6838-481c-881e-5c05cd843766)
Colour contrast 2,8:1... shall be 4,5:1 for number and 3:1 for text.
